docs/DeveloperGuide.md

```diff
@@ -561,10 +688,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 4.  The system should respond to user commands within 15 seconds.
 5.  The application should launch within 2 seconds.
 6.  The application should not store sensitive information (e.g., NRIC numbers) without explicit user consent.
-7.  The application should demonstrate stability during long-term usage without crashing or freezing.
+7.  The application should demonstrate stability during long-term usage like after a few hours without crashing or freezing.
```

Comment:
Perhaps we can change "during long-term usage like after a few hours" to "after 2 hours" to be more specific